chains/evm/calls/client/client.go

```diff
@@ -1,4 +1,4 @@
-package calls
+package client
```

Comment:
Not sure if this is the best package name @P1sar @mpetrun5

Reply:
i would leave it as it was, but probably would rename the file to calls. 
Any way this name looks wrong for me

Reply:
This probably will make more sence after we will rename calls package to something else. Like Contracts or something like that